source/install/self-managed-changelog.md

```diff
@@ -6,11 +6,73 @@ See the [changelog in progress](https://bit.ly/2nK3cVf) for the upcoming release
 
 Latest Mattermost Releases:
 
+- [Release v9.2 - Feature Release](#release-v9-2-feature-release)
 - [Release v9.1 - Feature Release](#release-v9-1-feature-release)
 - [Release v9.0 - Major Release](#release-v9-0-major-release)
 - [Release v8.1 - Extended Support Release](#release-v8-1-extended-support-release)
 - [Release v7.8 - Extended Support Release](#release-v7-8-extended-support-release)
 
+## Release v9.2 - [Feature Release](https://docs.mattermost.com/upgrade/release-definitions.html#feature-release)
+
+**Release day: November 16, 2023**
+
+### Important Upgrade Notes
+ - Fixed Data Retention Policies to run jobs when there is any "Custom retention policy" enabled even when the "Global retention policy" is set to "keep-forever". Before this fix, the Enabled Custom Policies wouldn't run as long as Global Policy was set to "keep-forever" or was disabled. After the fix, the custom policies will run automatically even when the Global Policy is set to "keep-forever". Once the server is upgraded, posts may unintentionally be deleted. Admins should make sure to disable all custom policies before upgrading, and then re-enable them again after upgrading.
```

Comment:
@vish9812 Is there any special testing that needs to be done to test this breaking change, or do the test steps in the Jira ticket cover everything https://mattermost.atlassian.net/browse/MM-44002?

Reply:
@amyblais Yes, those steps are pretty much it.